Rubberduck.VBEEditor/QualifiedModuleName.cs

```diff
@@ -96,15 +96,32 @@ public QualifiedMemberName QualifyMemberName(string member)
         public string ComponentName { get { return _componentName ?? string.Empty; } }
 
         public string Name { get { return ToString(); } }
+
         private readonly string _projectName;
+
+        public string ProjectName
+        {
+            get
+            {
+                return _projectName;
+            }
+        }
         private readonly string _projectPath;
 
+        public string ProjectPath
+        {
+            get
+            {
+                return _projectPath;
+            }
+        }
+
         public override string ToString()
         {
-            return _component == null && string.IsNullOrEmpty(_projectName)
+            return _component == null && string.IsNullOrEmpty(ProjectName)
                 ? string.Empty
-                : (string.IsNullOrEmpty(_projectPath) ? string.Empty : System.IO.Path.GetFileName(_projectPath) + ";")
-                     + _projectName + "." + _componentName;
+                : (string.IsNullOrEmpty(ProjectPath) ? string.Empty : System.IO.Path.GetFileName(ProjectPath) + ";")
+                     + ProjectName + "." + _componentName;
```

Comment:
Automatic refactoring in Visual Studio, my bad. I'll fix it with the raw input PR.